README.md

```diff
@@ -4,8 +4,12 @@ An AWS Lambda microservice presenting MIPS chart of accounts data
 ## Architecture
 
 This microservice is designed to retrieve a chart of accounts from a third-party API and present the data in a useful format.
-The initial implementation formats all accounts into a JSON string,
-future iterations will add formats to replace existing static files and filter out inactive accounts.
+
+Formats available:
+| Route | Description |
+| --- | --- |
+| /accounts.json | A dictionary mapping all active and inactive accounts to their friendly names. |
+| /tags.json | A list of valid values for either `CostCenter` or `CostCenterOther` tags. |
```

Comment:
Why did you use the term `tags`? 

The api doesn't need to know anything about `CostCenterOther`. It's just a list of valid cost center codes.    

Also i don't understand the difference between these two endpoints.

Reply:
I used the term `tags` because it's a list of valid tag values, similar to the json file in aws-infra. And it's not just a list of cost center codes, it's also the format for combining the codes and names into strings. Why do you say that the api doesn't need to know anything about `CostCenterOther` if we are going to use this api to determine valid tag values and our infrastructure partitions the values between the two tags? Also I kept the `accounts.json` api route after we talked about reducing down to a single route because I think it's useful for debugging to be able to see the data in a "raw" format before filtering and string formatting; but still consolidated the two tagging routes into a single route with a query string parameter.